### PR TITLE
Fix rubocop config and warnings

### PR DESCRIPTION
**Pull request by abcang**
Due to the version upgrade of rubocop, the number of new warnings has increased.

```
$ be rubocop --fail-level W --display-only-fail-level-offenses --parallel
Inspecting 723 files
.......................................................................................................................................................................................................................................................................................................W.............................................................................W.........W.......W..W................W................................................................................................................................................WW.................................................................................................................................................W...................

Offenses:

app/models/account.rb:390:5: W: Lint/MissingSuper: Call super to initialize state of the parent class.
    def initialize(account, attributes) ...
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/models/home_feed.rb:4:3: W: Lint/MissingSuper: Call super to initialize state of the parent class.
  def initialize(account) ...
  ^^^^^^^^^^^^^^^^^^^^^^^
app/models/list_feed.rb:4:3: W: Lint/MissingSuper: Call super to initialize state of the parent class.
  def initialize(list) ...
  ^^^^^^^^^^^^^^^^^^^^
app/models/poll.rb:75:5: W: Lint/MissingSuper: Call super to initialize state of the parent class.
    def initialize(poll, id, title, votes_count) ...
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/models/public_feed.rb:11:3: W: Lint/MissingSuper: Call super to initialize state of the parent class.
  def initialize(account, options = {}) ...
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/models/tag_feed.rb:15:3: W: Lint/MissingSuper: Call super to initialize state of the parent class.
  def initialize(tag, account, options = {}) ...
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/services/keys/claim_service.rb:10:5: W: Lint/MissingSuper: Call super to initialize state of the parent class.
    def initialize(account, device_id, key_attributes = {}) ...
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/services/keys/query_service.rb:9:5: W: Lint/MissingSuper: Call super to initialize state of the parent class.
    def initialize(account, devices) ...
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/services/keys/query_service.rb:22:5: W: Lint/MissingSuper: Call super to initialize state of the parent class.
    def initialize(attributes = {}) ...
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/paperclip/color_extractor.rb:145:10: W: Lint/FloatComparison: Avoid (in)equality comparisons of floats as they are unreliable.
      if s == 0.0
         ^^^^^^^^

723 files inspected, 10 offenses detected
```

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -2,7 +2,8 @@ require:
   - rubocop-rails
 
 AllCops:
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: 2.5
```
Comment (abcang) [inline .rubocop.yml:5]: The minimum supported version of Mastodon is currently 2.5.

File: .rubocop.yml [hunk 2]
```diff
@@ -2,7 +2,8 @@ require:
...
+  NewCops: disable
```
Comment (abcang) [inline .rubocop.yml:6]: New cops can be falsely detected, so I thought it wouldn't be necessary to actively enable them. The current unconfigured state is pending and the new cop remains disabled.
By setting it explicitly, the warning that prompts you to set the new cop will not be displayed.

File: app/models/account.rb [hunk 3]
```diff
@@ -385,15 +385,17 @@ def synchronization_uri_prefix
...
     def initialize(account, attributes)
-      @account     = account
-      @attributes  = attributes
-      @name        = attributes['name'].strip[0, string_limit]
-      @value       = attributes['value'].strip[0, string_limit]
-      @verified_at = attributes['verified_at']&.to_datetime
-      @errors      = {}
+      @original_field = attributes
+      string_limit = account.local? ? 255 : 2047
+      super(
```
Comment (abcang) [inline app/models/account.rb:393]: The same thing can be achieved by giving hash to the initializer of `ActiveModelSerializers::Model`.

File: app/models/account.rb [hunk 4]
```diff
@@ -415,22 +417,12 @@ def verifiable?
     end
 
     def mark_verified!
-      @verified_at = Time.now.utc
-      @attributes['verified_at'] = @verified_at
+      self.verified_at = Time.now.utc
+      @original_field['verified_at'] = @verified_at
```
Comment (abcang) [inline app/models/account.rb:421]: I had to maintain this process to propagate the changes to the fields of account.

File: app/models/public_feed.rb [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class PublicFeed < Feed
```
Comment (abcang) [inline app/models/public_feed.rb:3]: The methods defined in the Feed weren't used at all, so it didn't seem necessary to inherit them.

File: app/models/account.rb [hunk 6]
```diff
@@ -385,15 +385,17 @@ def synchronization_uri_prefix
...
-    attributes :name, :value, :verified_at, :account, :errors
+    attributes :name, :value, :verified_at, :account
 
     def initialize(account, attributes)
-      @account     = account
-      @attributes  = attributes
-      @name        = attributes['name'].strip[0, string_limit]
-      @value       = attributes['value'].strip[0, string_limit]
-      @verified_at = attributes['verified_at']&.to_datetime
-      @errors      = {}
```
Comment (abcang) [inline app/models/account.rb:396]: `errors` didn't seem to be used

**Review by ykzts — APPROVED** [review]

**Review by Gargron — APPROVED** [review]

**Pull request merged**